src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -26,11 +26,11 @@ public class SemanticQuickInfoSourceTests : AbstractSemanticQuickInfoSourceTests
         {
             using (var workspace = TestWorkspace.CreateCSharp(markup, options))
             {
-                await TestWithOptionsAsync(workspace, expectedResults);
+                await TestWithOptionsAsync(workspace, skipSpeculative: false, expectedResults);
             }
         }
 
-        private async Task TestWithOptionsAsync(TestWorkspace workspace, params Action<object>[] expectedResults)
+        private async Task TestWithOptionsAsync(TestWorkspace workspace, bool skipSpeculative, params Action<object>[] expectedResults)
```

Comment:
i'm a little confused as to why we would still need this.  Since quick-info is no longer using any speculative semantic models, there should be no effect of trying with/without speculation.  Indeed, i would be fine if the CanUseSpeculativeSemanticModelAsync part of the test was removed entirely (though keeping it would be fine.)

Reply:
Sorry about that. It was a left-over I should have cleaned up. Thanks